### PR TITLE
Nursery

### PR DESCRIPTION
**Pull request by toolslive**
Functionality passes the 1.x nursery tests.

### CODE REVIEW

File: src/client/remote_admin_test.ml [hunk 1]
```diff
@@ -113,11 +194,5 @@ let set_route_delta port () =
...
   __wrap__ port conversation
 
-let suite = 
-  let w f = Extra.lwt_bracket setup f teardown in
-  "nursery" >:::
-    ["set_interval" >:: w (set_interval 6666);
-     "get_fringe"  >:: w (get_fringe  5555);
-     "set_routing"  >:: w (set_route_delta  4444);
-    ]
+*)
```
Comment (NicolasT) [inline src/client/remote_admin_test.ml:197]: Should commented code be kept around?

File: src/hope/bstore.ml [hunk 2]
```diff
@@ -85,19 +85,22 @@ module BStore = (struct
         let rec _inner (tx: BS.tx) = function
           | Core.SET (k,v) ->
               begin
-                Lwtc.log "set: %s" k >>= fun () -> 
+                (* Lwtc.log "set: %s" k >>= fun () ->
```
Comment (NicolasT) [inline src/hope/bstore.ml:88]: Remove commented code (also below)?

OK to remove _check_interval?

File: src/hope/bstore.ml [hunk 3]
```diff
@@ -311,17 +334,92 @@ module BStore = (struct
...
+                  let start' = Some (last_key kvs_rev) in
+                  let mem' = List.fold_left 
+                    (fun mem (k,v) ->
+                      mem + String.length k + String.length v) mem kvs_rev 
+                  in
+                  if mem' < limit
+                  then loop start' mem' acc'
+                  else Lwt.return (acc', mem')
+              in
+              loop (Some "\xff") 0 []
```
Comment (NicolasT) [inline src/hope/bstore.ml:390]: Is starting at 0xff 'far enough to the right'?

File: src/hope/bstore.ml (hunk 2 above)
Comment (toolslive) [inline src/hope/bstore.ml:88]: the check interval indeed should be introduced, but only after we have something like a 'management sequence'.
so the introduction earlier was a bit premature.

File: src/hope/bstore.ml (hunk 3 above)
Comment (toolslive) [inline src/hope/bstore.ml:390]: nope. but for that I need to adapt baardskeerder.

File: src/client/remote_admin_test.ml (hunk 1 above)
Comment (toolslive) [inline src/client/remote_admin_test.ml:197]: These were tests from the 1.x era, and the suite was not part of the 2.x.  
So I re-enabled the suite and made the test I needed succeed again. If we encounter a bug in the other parts of the protocol, these could be used as guidance.

**Comment by NicolasT** [comment]
LGTM, but might be worth to put some of the dev TODO items in JIRA so they don't get forgotten?

**Pull request merged**